### ISSUE TITLE
Conda path is added $PATH when loading a RStudio project

### ISSUE THREAD

**Issue by ccasar** [rstudio/rstudio#10038]
### System details

    RStudio Edition : Desktop
    RStudio Version :  2021.09.0 Build 351
    OS Version      :  Manjaro Linux
    R Version       : 4.1.1

### Steps to reproduce the problem
Compare output of `Sys.getenv('PATH')` before and after loading a Rstudio project on a Linux system with conda installed. 

### Describe the problem in detail
Since updating from RStudio Desktop 1.4.1717 to 2021,09,0 when opening any project, `~/miniconda3/bin/` gets automatically added to the beginning of the PATH environment variable.

Before loading a project:
```
> Sys.getenv('PATH')
[1] "/home/christian/bin:/home/christian/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/rstudio/bin/postback"
```

After loading a project:
```
Sys.getenv('PATH')
[1] "/home/christian/miniconda3/bin:/home/christian/bin:/home/christian/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/rstudio/bin/postback"
```

This causes the installation of packages within the porject to fail because they try to use the libraries provided by conda instead of the system libraries, e.g. when installing the XML package.

If I downgrade back to RStudio 1.4.1717 this behavior of adding conda to PATH goes away. 

I did not find any mention of this behavior change in the RStudio release notes, so I assume it's a bug.

So far I could reproduce this on two different machines running Manjaro.

### Describe the behavior you expected
Do not automatically alter the PATH varariable or provide an option to disable it.

- [ x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ccasar** [comment]
It seems the problem was already reported in https://github.com/rstudio/rstudio/issues/9952

**Comment by kevinushey** [comment]
> This causes the installation of packages within the porject to fail because they try to use the libraries provided by conda instead of the system libraries, e.g. when installing the XML package.

Can you share the error message you saw?

**Comment by ccasar** [comment]
Sure:

```
> install.packages('XML')
Retrieving 'https://cloud.r-project.org/src/contrib/XML_3.99-0.8.tar.gz' ...
	OK [file is up to date]
Installing XML [3.99-0.8] ...
	FAILED
Error installing package 'XML':
===============================

* installing *source* package ‘XML’ ...
** package ‘XML’ successfully unpacked and MD5 sums checked
** using staged installation
  Cleaning up after installing the XML package
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking how to run the C preprocessor... gcc -E
checking for sed... /usr/bin/sed
checking for pkg-config... /usr/bin/pkg-config
checking for xml2-config... /home/christian/miniconda3/bin/xml2-config
USE_XML2 = yes
SED_EXTENDED_ARG: -E
Minor 9, Patch 10 for 2.9.10
Located parser file -I/home/christian/miniconda3/include/libxml2 -I/home/christian/miniconda3/include/parser.h
Checking for 1.8:  -I/home/christian/miniconda3/include/libxml2 -I/home/christian/miniconda3/include
Using libxml2.*
checking for gzopen in -lz... yes
checking for xmlParseFile in -lxml2... yes
You are trying to use a version 2.* edition of libxml
but an incompatible library. The header files and library seem to be
mismatched. If you have specified LIBXML_INCDIR, make certain to also
specify an appropriate LIBXML_LIBDIR if the libxml2 library is not in the default
directories.
ERROR: configuration failed for package ‘XML’
```

Note: I'm using R and RStudio desktop installed via the system repositories and there happens to be `xml2-config` installed within the conda base environment that is used here instead of the `xml2-config` provided by the system.

I tried setting `PATH` in the `.Renvrion` in my home directory but it still got overridden after loading a project which left my pretty confused until I found that other issue.

**Comment by kevinushey** [comment]
Thanks! I think this should now be installed in the latest [daily builds](https://dailies.rstudio.com), if you'd be willing to check.

The underlying motivation here was for users who had configured RStudio to use an Anaconda installation -- in that case, we wanted binaries associated with that Anaconda installation to be automatically made available to both the R console and RStudio terminals. However, changing the PATH in this way can evidently cause other issues.

In your case, I think we ended up over-zealously "activating" the Anaconda installation we discovered; that change has now been backed out.
